mm2src/mm2_tests.rs

```diff
@@ -8414,6 +8414,42 @@ fn test_get_public_key() {
         "022cd3021a2197361fb70b862c412bc8e44cff6951fa1de45ceabfdd9b4c520420"
     )
 }
+#[test]
```

Comment:
Please pay attention to spaces as @sergeyboyko0791 mentioned before here https://github.com/KomodoPlatform/atomicDEX-API/pull/1258#discussion_r854782747